src/modules/layoutUiRules/index.js

```diff
@@ -6,10 +6,12 @@ export {
     getAxisMinNumberOfDimsByVisType as getAxisMinNumberOfDimensions,
     getAxisPerLockedDimByVisType as getAxisPerLockedDimension,
     getAllLockedDimIdsByVisType as getAllLockedDimensionIds,
+    getDimensionMaxNumberOfItemsByVisType as getDimensionMaxNumberOfItems,
 } from './rulesHelper.js'
 
 export {
     hasAxisTooManyItemsByVisType as hasAxisTooManyItems,
+    hasDimensionTooManyItemsByVisType as hasDimensionTooManyItems,
```

Comment:
`hasDimensionTooManyItemsByVisType` doesn't sound correct to me. Should be `dimensionHasTooManyItemsByVisType`? I'm not a native english speaker either though so I'll loop in @jenniferarnesen 🙏

Reply:
I see that this might just be following the item above called `hasAxisTooManyItemsByVisType`, but that can't really be used as a reference as it was [added by me](https://github.com/dhis2/analytics/pull/161) (and I don't understand what I was thinking back then 🙈 )

Reply:
I guess it's following the convention from the line above? It's ugly, yeah, but consistent 😬

Reply:
Ok I blame 2019-Martin then 😬